eth/polar/backend.go

```diff
@@ -398,7 +393,7 @@ func (b *backend) GetLogs(
 // GetTd returns the total difficulty of a block in the canonical chain.
 // This is hardcoded to 69, as it is only applicable in a PoW chain.
 func (b *backend) GetTd(_ context.Context, hash common.Hash) *big.Int {
-	if header := b.polar.blockchain.GetHeaderByHash(hash); header != nil {
+	if header, err := b.polar.blockchain.GetHeaderByHash(hash); err == nil {
```

Comment:
doesn't match geth

Reply:
Just propogating error, useful for the client making call